docs/admin-guide.md

```diff
@@ -1,67 +1,54 @@
 ## Admin Guide
 
-- [Prerequisites](#prerequisites)
-- [Installation](#installation)
-- [Configuration](#configuration)
-- [Webhooks](#web-hooks)
-- [Slash Commands](#slash-commands)
-- [Onboard Users](#onboard-users)
-- [FAQ](#faq)
-- [Get Help](#get-help)
-
-## Prerequsites
+-   [Prerequisites](#prerequisites)
+-   [Installation](#installation)
+-   [Configuration](#configuration)
+-   [Webhooks](#web-hooks)
+-   [Slash Commands](#slash-commands)
+-   [Onboard Users](#onboard-users)
+-   [FAQ](#faq)
+-   [Get Help](#get-help)
 
 ## Installation
 
-### Marketplace Installation
-
-1. Go to **Main Menu > Plugin Marketplace** in Mattermost.
-2. Search for "Nagios" or find the plugin from the list.
-3. Select **Install**.
-4. When the plugin has downloaded and been installed, select **Configure**.
-
 ### Manual Installation
 
-If your server doesn't have access to the internet, you can download the latest [plugin binary release](https://github.com/mattermost/mattermost-plugin-nagios/releases) and upload it to your server via **System Console > Plugin Management**. The releases on this page are the same used by the Marketplace. To learn more about how to upload a plugin, see [the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
-
-### Cloud
-
-### Upgrade
-
-Ensure you configure the plugin before you enable it.
+You can download the latest [plugin binary release](https://github.com/mattermost/mattermost-plugin-nagios/releases) and upload it to your server via _System Console > Plugin Management_.
 
 ## Configuration
 
 ### Enter the URL for your Nagios instance
 
-1. In Mattermost, go to **System Console > Plugins > Nagios**.
-2. Set the **Nagios URL**. Remember to add `http://` or `https://` at the beginning (e.g.: `https://nagios.fedoraproject.org`).
-3. Set **Nagios Username** to the Nagios user used to authentiacte.
-4. Sett **Nagios Password** to the password of that user.
-5. Select **Save**.
-6. Go to **Plugins > Nagios**, then select **Enable**.
+1. In Mattermost, go to _System Console > Plugins > Nagios_.
```

Comment:
I think the original bold syntax is easier to read. Same with every other italics that was introduced I would say

```suggestion
1. In Mattermost, go to **System Console > Plugins > Nagios**.
```